mne/io/eeglab/tests/test_eeglab.py

```diff
@@ -77,7 +92,6 @@ def test_io_set():
     shutil.copyfile(op.join(base_dir, 'test_epochs.fdt'),
                     op.join(temp_dir, 'test_epochs.dat'))
     with warnings.catch_warnings(record=True) as w:
-        warnings.simplefilter('always')
```

Comment:
sorry posted at the wrong place. See: https://github.com/mne-tools/mne-python/pull/2975/files#r55546078